### PR TITLE
BuildKite support for shared_ci

### PR DESCRIPTION
**Pull request by gkassabli**
Files to support BuildKite runs on shared_ci repo.

### CODE REVIEW

**Comment by gkassabli** [comment]
Buildkite is green!

File: .buildkite/premerge.definition.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+agent_queue_id: trigger-pipelines
+description: Unity GDK Shared CI repository pre-merge CI pipeline.
```
Comment (jamiebrynes7) [inline .buildkite/premerge.definition.yaml]: `GDK for Unity` 😉

**Review by jamiebrynes7 — APPROVED** [review]
LGTM module small change

File: .buildkite/premerge.definition.yaml (hunk 1 above)
Comment (gkassabli) [reply .buildkite/premerge.definition.yaml]: Oops, will fix this!

**Pull request merged**